Dockerfile

```diff
@@ -43,6 +43,9 @@ WORKDIR /staging
 # Copy main executable to staging area
 RUN cp "$(swift build --package-path /build -c release --show-bin-path)/Run" ./
 
+# Copy static swift backtracer binary to staging area
+RUN cp "/usr/libexec/swift/linux/swift-backtrace-static" ./
```

Comment:
@finestructure I don't think this is enough. Down below you should also need https://github.com/vapor/template/pull/120

Reply:
We're not setting this via the Dockerfile so we can update it without having to rebuild. Our full `SWIFT_BACKTRACE` is

`enable=yes,images=all,interactive=no,swift-backtrace=./swift-backtrace-static,threads=all,output-to=stderr`

Reply:
You are settings this via the Dockerfile though, down below. Maybe that's unintended?

Reply:
to be clear, by "this" I mean "SWIFT_BACKTRACE".

Reply:
Oh, I'd missed that! In my testing this is being overridden by our dynamically set variable but I'll definitely want to remove it to avoid confusing which one is being used. Good catch, thanks!